spec/unit/ace/transport_app_spec.rb

```diff
@@ -308,6 +318,59 @@ def app
       end
     end
 
+    context 'when given flag debug' do
+      it 'returns 200 with success' do
```

Comment:
you probably want to add:

```
expect(last_response).to be_ok
expect(last_response.status).to eq(200)
expect(last_response.status).to eq(200)
```

back in

Reply:
Done